tests/nodeos_high_transaction_test.py

```diff
@@ -218,6 +222,20 @@ def findTransInBlock(transId, transToBlock, node):
     #verify nodes are in sync and advancing
     cluster.waitOnClusterSync(blockAdvancing=5)
 
+    nodeOrder = None
```

Comment:
While iteration of `nodeOrder` on line 283 is protected with another `if args.send_duplicates:`, better to initialize it to `[]` rather than `None`.  Later logic changes that might remove the check would then degrade gracefully to doing nothing in the loop rather than throwing `TypeError`.